packages/teleterm/src/sharedProcess/sharedProcess.ts

```diff
@@ -1,13 +1,24 @@
-import { Server } from '@grpc/grpc-js';
+import { Server, ServerCredentials } from '@grpc/grpc-js';
 
 import createLoggerService from 'teleterm/services/logger';
-import { getServerCredentials } from 'teleterm/services/grpcCredentials';
+import {
+  createInsecureServerCredentials,
+  createServerCredentials,
+  generateAndSaveGrpcCert,
+  GrpcCertName,
+  readGrpcCert,
+  shouldEncryptConnection,
+} from 'teleterm/services/grpcCredentials';
 import { RuntimeSettings } from 'teleterm/mainProcess/types';
 import Logger from 'teleterm/logger';
 
 import { PtyHostService } from './api/protogen/ptyHostService_grpc_pb';
 import { createPtyHostService } from './ptyHost/ptyHostService';
 
+const runtimeSettings = getRuntimeSettings();
+initializeLogger(runtimeSettings);
+initializeServer(runtimeSettings);
+
```

Comment:
This was just moved from the bottom of the file to the top, before function declarations so that you can immediately see what's going to be executed when this file is loaded.